winlogbeat/_meta/beat.full.yml

```diff
@@ -12,13 +12,13 @@
 # The registry file is where Winlogbeat persists its state so that the beat
 # can resume after shutdown or an outage. The default is .winlogbeat.yml
 # in the directory in which it was started.
-#winlogbeat.registry_file: .winlogbeat.yml
+winlogbeat.registry_file: .winlogbeat.yml
 
 # Diagnostic metrics that can retrieved through a web interface if a
 # bindaddress value (host:port) is specified. The web address will be
 # http://<bindaddress>/debug/vars
-#winlogbeat.metrics:
-#  bindaddress: 'localhost:8123'
+winlogbeat.metrics:
```

Comment:
If it is not enabled by default and does not have an enabled flag, it should be commented out. Otherwise short and full config have different behaviour.